runners/core-java/src/main/java/org/apache/beam/runners/core/ReduceFnRunner.java

```diff
@@ -783,7 +897,7 @@ private void emitIfAppropriate(ReduceFn<K, InputT, OutputT, W>.Context directCon
   /**
    * Do we need to emit a pane?
    */
-  private boolean needToEmit(boolean isEmpty, boolean isFinished, PaneInfo.Timing timing) {
+  private boolean needToEmitPane(boolean isEmpty, boolean isFinished, PaneInfo.Timing timing) {
```

Comment:
To be honest, the term "pane" misleads users pretty often - they don't realize that they are "just elements" so expect things like `Combine.PerKey()` to have a unique value per key even in the presence of multiple "panes". So I'd rather not propagate it, but rather start to eliminate it where we can.